DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/use_library/part2_lib_func_call/CMakeLists.txt

```diff
@@ -0,0 +1,343 @@
+
+# Direct CMake to use icpx rather than the default C++ compiler/linker on Linux
+# and icx-cl on Windows
+if(UNIX)
+    set(CMAKE_CXX_COMPILER icpx)
+else() # Windows
+    include (CMakeForceCompiler)
+    CMAKE_FORCE_CXX_COMPILER (icx-cl IntelDPCPP)
+    include (Platform/Windows-Clang)
+endif()
+
+cmake_minimum_required (VERSION 3.7.2)
+
+project(use_library CXX)
+
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+
+###############################################################################
+### Customize these build variables
+###############################################################################
+set(SOURCE_FILES src/use_library.cpp)
+set(TARGET_NAME use_library)
+set(LIB_FILENAME lib_rtl)
+
+# Library source files
+set(RTL_C_MODEL src/${LIB_FILENAME}_model.cpp)
+set(RTL_SPEC src/${LIB_FILENAME}_spec.xml)
+set(RTL_V src/${LIB_FILENAME}.v)
+if (WIN32)
+    set(RTL_SOURCE_OBJECT ${LIB_FILENAME}.obj)
+    set(LIBRARY_ARCHIVE ${LIB_FILENAME}.lib)
+else()
+    set(RTL_SOURCE_OBJECT ${LIB_FILENAME}.o)
+    set(LIBRARY_ARCHIVE ${LIB_FILENAME}.a)
+endif()
+
```

Comment:
i do not know CMake as well as Yohann, we should work with him to update the output so the blue text shows the calls to `fpga_crossgen` and `fpga_libtool`.